gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
+            components are present in this section, however React hooks are documented here in addition to JavaScript
+            functions and TypeScript types.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Guidelines</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Rather than documentation of specific components or usable pieces of code, this section includes higher level
+            descriptions, tutorials, and recommendations regarding usage of RSC components and web UI development in
+            general.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+      </NxDescriptionList>
       <NxP>
-        Documentation of all React components available to consumers of RSC, including demonstrations, example
-        code, and full documentation of relevant React <NxCode>props</NxCode>. Examples here
-        demonstrate basic usage of each component along with some modicum of supporting code - for example,
-        code tracking the toggle state of a <NxCode>NxCheckbox</NxCode>. This supporting code
-        should be taken as example only. Depending on the architectural needs of the consuming application, there
-        are many different ways that state management and other supporting concerns could be handled.
+        In addition to the groupings within the sidebar, you will notice that each page entry has an icon which
+        indicates the type of item documented on that page:
       </NxP>
-      <NxH4>Guidelines</NxH4>
-      <NxP>Miscellaneous advice for RSC consumers</NxP>
-      <NxH4>Styles - HTML Elements</NxH4>
-      <NxP>
-        Demonstrations of raw CSS classes that are available to consumers of RSC. Some of these classes are also
-        available as React components, but are documented here as well for lower-level/non-React use.
-      </NxP>
-      <NxH4>Styles - Mixins &amp; Helpers</NxH4>
-      <NxP>
-        Documentation of SCSS mixins and CSS helper classes available to consumers of RSC.
-      </NxP>
-      <NxH4>Layout Examples</NxH4>
-      <NxP>
-        Explanations of broader layout concepts that includes groups of components and/or styles working together.
-      </NxP>
-      <NxH4>JavaScript &amp; TypeScript Utilities</NxH4>
-      <NxP>Utility function and datatypes available to users of RSC.</NxP>
+      <NxDescriptionList>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faReact} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Documentation for a React component or family of closely related React components. Typically includes
+            technical details about the props that may be passed to the component along with live examples of its use.
```

Comment:
This section includes documentation for a React component or a family of related React components. It contains technical details with examples, on the properties that can be passed to the component.